test/torchaudio_unittest/compliance_kaldi_test.py

```diff
@@ -56,11 +55,8 @@ class Test_Kaldi(common_utils.TempDirMixin, common_utils.TorchaudioTestCase):
     def setUp(self):
         super().setUp()
 
-        # test signal for testing resampling
-        self.test_signal_sr = 16000
-        self.test_signal = common_utils.get_whitenoise(
-            sample_rate=self.test_signal_sr, duration=0.5,
-        )
+        # test audio file corresponding to saved kaldi ark files
+        self.test_filepath = common_utils.get_asset_path('kaldi_file_8000.wav')
```

Comment:
Is this for lint issue? I see that `test_filepath` attribute is already defined at class level. So this appears redundant and it seems that we can get rid of the whole `setUp` method.